### PR TITLE
HH

### PR DESCRIPTION
**Pull request by Illia-Boiko**
- [DEMO LINK](https://Illia-Boiko.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Illia-Boiko.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,21 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+    data-qa="big"
+    class="big_s">
+      <input type="text"
+      data-qa="keypress"
+      class="big__s"
+      placeholder="Try “Los Angeles”">
```
Comment (danheim) [inline src/index.html:20]: https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping

**Review by danheim — CHANGES_REQUESTED** [review]
Fix formatting

**Comment by Illia-Boiko** [comment]
- [DEMO LINK](https://Illia-Boiko.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Illia-Boiko.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+html {
+  font-family: Avenir, serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+}
+
+.big__s {
```
Comment (danheim) [inline src/style.css:22]: big__search
не сокращай слова

**Review by danheim — APPROVED** [review]

**Pull request closed without merging**